lib/active_model/serializer/reflection.rb

```diff
@@ -17,7 +17,7 @@ class Serializer
     #
     # So you can inspect reflections in your Adapters.
     #
-    Reflection = Struct.new(:name, :options) do
+    Reflection = Struct.new(:name, :options, :proc) do
```

Comment:
I prefer `callable` to refer to a `callable` when, like here, it can be a block, proc, or lambda